include/nbl/video/ILogicalDevice.h

```diff
@@ -423,6 +423,8 @@ class NBL_API2 ILogicalDevice : public core::IReferenceCounted, public IDeviceMe
         //
         virtual void unmapMemory(IDeviceMemoryAllocation* memory) = 0;
 
+        virtual void* getExternalMemoryHandle(IDeviceMemoryBacked* obj) const = 0;
```

Comment:
do we need it though?

We already have `getExternalMemoryHandle` on the `IDeviceMemoryBacked` itself

Reply:
IDeviceMemoryBacked calls this one and caches the returned handle. This could be removed. Also, I think these handles need to be released by platform specific functions such as CloseHandle on win32 so we might be needing to track them.